addons/web/static/tests/helpers/utils.js

```diff
@@ -855,7 +855,7 @@ export async function dragAndDrop(from, to, position) {
  *
  * @param {Element | string} from
  */
-export async function drag(from) {
+export async function drag(from, pointerType = "mouse") {
```

Comment:
I added the possibility to set a custom pointerType (touch of course in my tests)